tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es2020",
```

Comment:
It probably does not make a huge difference, but I've seen you use `ecmaVersion: 2021` in [another PR](https://github.com/vector-im/element-desktop/pull/197/files?file-filters%5B%5D=.js&file-filters%5B%5D=.lock#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R9)
Which target are we aiming for?

Reply:
So far, my approach has been to generally allow for all modern / on the standards track syntax where possible, but the reason you're seeing different values is because of different environments:

* In this repo, we're targeting Node LTS (currently 14), and ES 2020 is what's supported there
* In the `element-desktop` PR you linked, we're targetting Electron, so we can use exactly whatever V8 supports over there, which seems to be quite modern, so I've set 2021 to match [compat table results](https://kangax.github.io/compat-table/es2016plus/)
* In other repos like JS SDK, React SDK, and Element Web, the available syntax is gated by what Babel will accept and compile for our supported browsers